### DISCUSSION TITLE
自分がフォローしているOrganizationを確認する手段がなさそう

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#121] [ヘルプ]
2021年冬に行なわれたアップデートで、Organizationをフォローする機能が追加されました。[^1]
ところが、自分がフォローしているOrganizationを確認できるページやAPIがなさそうな気がします。
自分が気づいていないだけで、どこかにあるのでしょうか？
ないのであれば、あると嬉しいです。

[^1]: https://blog.qiita.com/organization-update/

**Comment by kskwtnk** [comment]
@munierujp 
ご意見ありがとうございます！

まず、現状はページもAPIも存在していません……申し訳ないです :bow:

こちらのブログで今後のUIアップデートについて取り扱っていまして、明確に記載していないのですがOrganization一覧を検討している最中でした

https://blog.qiita.com/ui-update-plan-for-2022/

提供する形式や時期が未定なのですが、Organizationへの動線は作成したいと思っています！
